test/livebook/session/data_test.exs

```diff
@@ -48,6 +48,29 @@ defmodule Livebook.Session.DataTest do
     end
   end
 
+  describe "apply_operation/2 given :set_notebook_attributes" do
+    test "returns an error given an unknown attribute key" do
+      data = Data.new()
+
+      attrs = %{unknown: :value}
+      operation = {:set_notebook_attributes, self(), attrs}
+
+      assert :error = Data.apply_operation(data, operation)
```

Comment:
What does the UI do in this case? Just fails to import it? Perhaps we should print an error message?

Reply:
This isn't used for importing, it's just an operation similar to `:set_cell_attributes` that updates the session data we have. These attributes are usually updated via UI in one way or another and we control what these are. Technically we could raise an error, though returning `:error` this is more in line with all other errors.